src/actions/pause/fetchScopes.js

```diff
@@ -64,13 +64,13 @@ export function fetchScopes() {
       frame.generatedLocation.sourceId
     );
 
-    if (generatedSourceRecord.get("isWasm")) {
+    if (generatedSourceRecord && generatedSourceRecord.get("isWasm")) {
       return;
     }
 
     const sourceRecord = getSource(getState(), frame.location.sourceId);
 
-    if (sourceRecord.get("isPrettyPrinted")) {
```

Comment:
I'd like to figure out why these sources aren't around when we're doing `fetchScopes`